pkg/discovery/servicehandler.go

```diff
@@ -520,3 +523,12 @@ func isPublished(api *anypoint.API, authPolicy string, c cache.Cache) (bool, str
 		return true, checksum
 	}
 }
+
+func getMapFromInterface(generic interface{}) map[string]interface{} {
```

Comment:
not sure if I sound picky, but may be the variable 'generic' could be called something else